src/senselab/utils/model_utils.py

```diff
@@ -0,0 +1,160 @@
+"""This module provides utility classes for handling common utilities based on model type."""
```

Comment:
this is HF-specific. not every framework is structured with tokenizers, feature_extractor, models, and pipelines. I still wonder if we really need all this or if it's just an over-complication. please, help me understand your choice

Reply:
Thank you for your feedback. The plan was to have functions like `get_classifier` (`get_pipeline` now) and `get_tokenizer` to create a standardized way to access these components across different utils, making it easier to work with various models. You’d just call `model_utils.get_classifier()` to get the classifier and `model_utils.get_tokenizer()` to fetch the tokenizer for the model. If tokenizer is not available for a model and a long text is passed that exceeds the token limit, we could show an error. But I understand your concern about complexity and can rework the entire thing to work as it does currently with if-else checks at the `api.py` level.